tests/test_plotting.py

```diff
@@ -137,3 +137,26 @@ def test_live_plot(mock: MagicMock) -> None:
     assert mock.call_args_list[2].args[2] == 0.0
     assert mock.call_args_list[2].args[4] == 153
     assert mock.call_args_list[2].args[5] == 1.0
+
+
+@patch("RATapi.utils.plotting.plot_ref_sld_helper")
+def test_plot_ref_sld(mock: MagicMock, request) -> None:
+    problem = request.getfixturevalue("input_project")
+    results = request.getfixturevalue("reflectivity_calculation_results")
```

Comment:
The strongly recommended way to request fixtures is to include the them in the function arguments. This method of requesting fixtures is for when the choice of fixture depends on other parameters. I've seen that line 554 of "test_run.py" has a line like this as well (no idea who wrote it . . .) - could you please also change that to include the fixture in the function argument.